src/main/java/com/kodilla/ecommercee/controller/UserController.java

```diff
@@ -13,20 +16,31 @@
 public class UserController {
 
     @Autowired
-    private TokenService dbService;
+    private TokenService tokenService;
+
+    @Autowired
+    private UserService service;
+
+    @Autowired
+    private UserMapper userMapper;
 
     @RequestMapping(method = RequestMethod.POST, value = "createUser", consumes = APPLICATION_JSON_VALUE)
-    public UserDto createUser(@RequestBody UserDto userDto) {
-        return new UserDto(1L, "testUser", "1", null);
+    public void createUser(@RequestBody UserDto userDto) {
+        service.saveUser(userMapper.mapToUser(userDto));
     }
 
     @RequestMapping(method = RequestMethod.PUT, value = "blockingUser")
     public UserDto blockingUser(@RequestParam Long userId) {
-        return new UserDto(1L, "testUser", "0", null);
+        UserDto userDto = userMapper.mapToUserDto(service.getUser(userId));
```

Comment:
Nie ma konieczności pobierania usera, mapowania, ustawiania wartości w dto, kolejnego mapowania i zapisywania. Jest zbyt dużo operacji a można to wykonać odpowiednim zapytaniem w repository (metoda w serwisie oczywiście powinna istnieć) - https://www.baeldung.com/spring-data-jpa-query